.github/workflows/release-facebook.yml

```diff
@@ -0,0 +1,105 @@
+name: Release Facebook
+
+on:
+  workflow_dispatch:
+    inputs:
+      page_id:
+        description: 'Path parameter for /v19.0/{page-id}/photos endpoint'
+        required: false
```

Comment:
Could you make parameters acceptable from both inputs and vars/secrets?

Refer to https://github.com/mseninc/blog/blob/release/.github/workflows/release-02-preprocess.yml

Reply:
Okay.

Reply:
I added the parameters to the `workflow_dispatch` inputs and made the value to accept inputs and vars/secrets.
I used `::add-mask::` to mask the secret values (user access token and page access token).
I cannot yet test to input the values in the actions page.

Reply:
I didn't know `::add-mask::` feature.

Thanks. Ah since it's not yet merged.
Do you think I should merge this?

Reply:
I restored the `if` conditions to check hash, and removed this branch from `on` triggers, so now it could be merged.
It does not conflict with other code.